apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -22,12 +27,37 @@ h2 + .c-card-wrapper {
     padding: 0;
 }
 
+.c-card-list--fillContainer {
+    grid-template-columns: 1fr;
+
+
+    @include f.media('>=mid') {
+        grid-template-columns: auto auto auto;
```

Comment:
would `repeat(auto, 3)` or similar work here?

Reply:
would `repeat(1fr, 3)` or similar achieve the same?

Reply:
Sadly that doesn't seem to work but I'll keep looking for alternatives